content/pages/office-policy.md

```diff
@@ -0,0 +1,50 @@
+Title: Office Policy
+Category: pages
+Status: hidden
+Slug: office-policy
+
+(Version 2.0 March 8th 2015)
+
+## Personnel ##
+
+* If the office is open for office hours, drop-ins or for any other reason, a
+**minimum** of **one** committee member of WiCS must be present in the office at
+all times.
+* All WiCS members occupying the office are responsible for complying with and
+enforcing the Code of Conduct.
+
+## Cleanup ##
+
+* The last WiCS staff member to occupy the office is responsible for cleanup.
+* If the recycling and/or garbage is full, empty it.
+* **No perishable food items are to be left in the office.**
+* Ensure no hardware is left on the windowsill.
+* Ensure the chairs are stacked next to the metal cabinet.
+* Ensure there are no stray cables on the floor.
+* Ensure no garbage is left on the desks, chairs, shelves or floor.
+* Ensure the lights are turned off.
+* Lock the door.
+
+## Office Hours ##
+
+* Office hours and services are open only to ciswomen, transwomen and non-binary
+ folks.
```

Comment:
Also one last detail: this line should read "cis women, trans women, and non-binary individuals." to be consistent with the rest of our content.